packages/codec-components/src/react/components/codec/format.errors.uint-padding-error.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import type { Format } from "@truffle/codec";
+import { createCodecComponent } from "../../utils/create-codec-component";
+
+export const { UintPaddingError } = createCodecComponent(
+  "UintPaddingError",
+  // TODO
+  (data: Format.Errors.UintPaddingError) => <span>{data.raw}</span>
```

Comment:
All these errors with TODO won't indicate "error" anywhere, right? Maybe we should prefix these so it's clear when it's an error case

Reply:
Done.